zebra/zebra_evpn_neigh.c

```diff
@@ -175,8 +175,7 @@ void zebra_evpn_install_neigh_hash(struct hash_bucket *bucket, void *ctxt)
 	n = (struct zebra_neigh *)bucket->data;
 
 	if (CHECK_FLAG(n->flags, ZEBRA_NEIGH_REMOTE))
-		zebra_evpn_rem_neigh_install(wctx->zevpn, n,
-					     false /*was_static*/);
+		zebra_evpn_rem_neigh_install(wctx->zevpn, n, false /*was_static*/);
```

Comment:
clang-format disagrees?

Reply:
Hi @mjstapp , Thanks for your time in reviewing the changes.
In many places the clang-format is not proper, for example in the above code snippet the params can be includuded in the same line but it got devided and giving in multiple lines. Am just trying to correct this only on the places where can be corrected, remaining all other places am keeping as it is. thanks

Reply:
I think the issue will be that we're using clang-format and our config file for it as a reference for now. Diverging from it, other than for a couple of well-known patterns such as cli definition blocks, is going  to be confusing.